### DISCUSSION TITLE
Maizzle + Tailwindcss/typography doesn't work

### DISCUSSION THREAD

**Discussion by cowboycodr** [tailwindlabs/tailwindcss#13489] [Help]
Maizzle struggles to generate the proper inline-styles when using the typography plugin. It's actually a huge headache. The closest solution I've found is #7684. Except, this was ~2 years ago, and the solution doesn't even seem to exist anymore.

Does anyone have any ideas how I can get typography working with Maizzle?

**Comment by brandonmcconnell** [comment]
Have you reached out to them directly on their repo? https://github.com/maizzle/framework

Another option to consider is [react.email](http://react.email), which has a built-in Tailwind CSS compiler complete with config. I tried installing the typography plugin in a test instance earlier locally when I first saw your question in Discord, without any luck, but I reached out to @zenorocha on your behalf about it on Twitter here: https://x.com/branmcconnell/status/1777846575836106947

**Reply by cowboycodr** [reply]
Thank you for suggesting to reach out directly to the Maizzle team and for trying the typography plugin with react.email. Indeed, I've already contacted the Maizzle team about this issue. They confirmed that the complexity of [Tailwind's typography plugin makes it particularly challenging to use for inline styles](https://maizzle.com/guides/markdown-emails#tailwindcsstypography).

Additionally, I've explored using react.email in the past. However, I encountered limitations regarding their support for Tailwind plugins, which seems minimal if present at all. Your Twitter thread essentially confirms this.

I've been working to resolve this for about two weeks now. Your effort to help by reaching out to @zenorocha is greatly appreciated. I'll be sure to keep you updated if I come across a solution that works.

**Reply by brandonmcconnell** [reply]
I saw the same. Disappointing to hear, but hopefully they'll find a way to support plugins soon.

Thanks, keep me posted. Best of luck!

**Reply by cowboycodr** [reply]
If you're still interested I found a hellish solution.

**Reply by brandonmcconnell** [reply]
@cowboycodr I'm intrigued. Tell me. :)
